charts/spire/charts/spire-server/values.yaml

```diff
@@ -487,6 +487,10 @@ notifier:
     enabled: true
     ## @param notifier.k8sBundle.namespace Namespace to push the bundle into, if blank will default to SPIRE Server namespace
     namespace: ""
+    ## @param notifier.k8sBundle.apiServiceLabel If set, rotate the CA Bundle in API services with this label set to true.
+    apiServiceLabel: ""
```

Comment:
Can we make these bools, defaulted to false?

Reply:
These are the "key" part of the label. Say 

`apiServiceLabel: "example.com/spiffe"`. Then the actual label on an ApiService has to be `"example.com/spiffe": "true"`.

Reply:
OHH. That makes more sense.

When webhooks are deployed by the chart, does it make sense to always do the rotation automatically?

Reply:
I am too new to spiffe to answer this question. I see that networkservicemesh project uses this features using their static manifest. I am just trying to reproduce the same behavior using helm chart.

Reply:
Ok, thanks. I'll ask around. Will try and get back to you soon.

Reply:
Ok, so it sounds like those options were added primarily for the predecessor of spire-controller-manager. Spire-controller-manager will do it itself, so having the options aren't bad, but not needed for proper functioning of the spire-controller-manager. So, I think we're good the way the PR is.